lib/commands/performance.js

```diff
@@ -112,8 +117,8 @@ commands.mobileStartPerfRecord = async function mobileStartPerfRecord (opts = {}
       args.push('-p', pid);
     }
   }
-  const proc = new SubProcess('instruments', args);
-  log.info(`Starting 'instruments' with arguments: ${args.join(' ')}`);
+  const proc = new SubProcess(instrumentsPath, args);
+  log.info(`Starting ${instrumentsPath} with arguments: ${args.join(' ')}`);
```

Comment:
This should still be in single quotes, to more easily see the value and any problems in the path.

Reply:
fixed